src/content/en/updates/2018/09/chrome-70-media-updates.md

```diff
@@ -0,0 +1,128 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 70: Cross-codec and cross-bytestream buffering and playback, Opus in MP4 with MSE, and protected content playback allowed by default on Android.
+
+{# wf_updated_on: 2018-09-17 #}
+{# wf_published_on: 2018-09-17 #}
+{# wf_tags: news,chrome70,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: A round up of the audio/video updates in Chrome 70: Cross-codec and cross-bytestream buffering and playback, Opus in MP4 with MSE, and protected content playback allowed by default on Android. #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 70 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- [Cross-codec and cross-bytestream buffering and
+  playback](#sourcebuffer-changetype) is possible in MSE.
+- Chrome now supports [Opus in MP4 with MSE](#opus-in-mp4-for-mse).
+- Protected content playback is [allowed by
+  default](#protected-content-allowed-by-default) on Android.
+
+## Support for codec and container switching in MSE {: #sourcebuffer-changetype }
+
+Chrome is [adding support] for improved cross-codec or cross-bytestream
+transitions in [Media Source Extensions] (MSE) playback using a new
+`changeType()` method on `SourceBuffer`. It basically allows the type of media
+bytes subsequently appended to the `SourceBuffer` to be changed.
+
+While the [W3C REC version of MSE] supports adaptive playback of media, that
+adaptation requires that any media appended to a `SourceBuffer` must conform to
```

Comment:
I'm having trouble parsing the second third of the sentence. My suggestion involves all three parts.

The current version of the spec supports adaptive playback; however, adaptive playback is restricted to the source buffer's initial MIME type.  Codecs from that type...